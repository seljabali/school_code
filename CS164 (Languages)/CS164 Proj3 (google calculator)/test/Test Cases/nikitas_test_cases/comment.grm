//This is a comment that should be ignored
%left '+' //Here is another comment
%% //And another
A -> A '+' B
//And yet another comment
   | B ; //So many comments
B -> /[0-9]+/ ;
  //   Everywhere you look // yaya
C -> /[a-z]/;
