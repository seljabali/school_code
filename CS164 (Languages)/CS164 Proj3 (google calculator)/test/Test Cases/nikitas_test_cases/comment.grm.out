(Grammar
    (Declarations
        (assoc \+ 0 left)
    )
    (Rules
        (Rule A
            (production [ A, \+, B, ]
            )
            (production [ B, ]
            )
        )
        (Rule B
            (production [ [0-9]+, ]
            )
        )
        (Rule C
            (production [ [a-z], ]
            )
        )
    )
    (StartSymbol A)
)
